config/crd/bases/workload.nephio.org_amfdeployments.yaml

```diff
@@ -0,0 +1,366 @@
+                      type: string
+                    uid:
+                      description: 'UID of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids'
+                      type: string
+                  type: object
+                  x-kubernetes-map-type: atomic
+                type: array
+              interfaces:
+                description: Interfaces defines the interfaces associated with the
+                  NF deployment
+                items:
+                  description: InterfaceConfig defines the configuration of the interface
+                  properties:
+                    ipv4:
+                      description: IPv4Config defines the ipv4 configuration of the
+                        interface
+                      properties:
+                        address:
+                          description: Address defines the IPv4 address and prefix
+                            length in CIDR notation [IP prefix, range IPv4 with host
+                            bits]
+                          type: string
+                        gateway:
+                          description: Gateway defines the IPv4 address associated
+                            to the interface as a gateway
+                          type: string
+                      required:
+                      - address
+                      type: object
+                    ipv6:
+                      description: IPv6Config defines the ipv6 configuration of the
+                        interface
+                      properties:
+                        address:
+                          description: Address defines the IPv6 address and prefix
+                            length in CIDR notation [IP prefix, range IPv6 with host
+                            bits]
+                          type: string
+                        gateway:
+                          description: Gateway defines the IPv6 address associated
+                            to the interface as a gateway
+                          type: string
+                      required:
+                      - address
+                      type: object
+                    name:
+                      description: Name defines the name of the interface
+                      maxLength: 253
+                      minLength: 1
+                      type: string
+                    vlanID:
+                      description: VlanID defines the specific vlan id associated
+                        on this interface
+                      type: integer
+                  required:
+                  - name
+                  type: object
+                type: array
+              networkInstances:
+                description: NetworkInstances defines the network instances associated
```

Comment:
Guess we discussed this earlier, but good to comment if the VRF in question is part of the virtual routing/forwarding fabric, i.e. cluster/worker node/gateway, or the NF itself ?..good to comment and clarify how this differentiate from the NADs which is NF function specific..

Reply:
each NF: AMF, SMF, UPF inherits the same nf deploy types, so this is the reason. This ensures the code reuse.